avocado/core/sysinfo.py

```diff
@@ -161,8 +162,8 @@ def _set_collectibles(self):
             log.debug(e.args[0])
 
     def _get_installed_packages(self):
-        sm = software_manager.SoftwareManager()
-        installed_pkgs = sm.list_all()
+        dpkg_backend = dpkg.DpkgBackend()
+        installed_pkgs = dpkg_backend.list_all()
```

Comment:
Before that change, IIUC, we had a "generic solution" without specifying the backend. And here you are using dpkg. What happens if we call this on a non dpkg system?

Reply:
Right, unless I'm missing something, this is pinning the dpkg backend, no matter what the current system uses.  I don't understand this change.